content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1109 @@
+  - matchLabels:
+      rbac.example.com/aggregate-to-monitoring: "true"
+rules: [] # コントロールプレーンは自動的にルールを入力します
+```
+既存の集約されたClusterRoleのラベルセレクターと一致する新しいClusterRoleを作成すると、その変更をトリガーに、集約されたClusterRoleに新しいルールが追加されます。
+`rbac.example.com/aggregate-to-monitoring: true`ラベルが付けられた別のClusterRoleを作成して、ClusterRole「monitoring」にルールを追加する例を以下に示します。
+
+
+Here is an example that adds rules to the "monitoring" ClusterRole, by creating another
+ClusterRole labeled `rbac.example.com/aggregate-to-monitoring: true`.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: monitoring-endpoints
+  labels:
+    rbac.example.com/aggregate-to-monitoring: "true"
+# ClusterRole「monitoring-endpoints」を作成すると、
+# 以下のルールがClusterRole「monitoring」に追加されます
+rules:
+- apiGroups: [""]
+  resources: ["services", "endpoints", "pods"]
+  verbs: ["get", "list", "watch"]
+```
+
+[デフォルトのユーザー向けRole](#デフォルトroleとclusterrolebinding)はClusterRoleの集約を使用します。これによりクラスター管理者として、 デフォルトroleを拡張するため、{{< glossary_tooltip term_id="CustomResourceDefinition" text="CustomResourceDefinitions" >}}または集約されたAPIサーバーなどによって提供されたルールをカスタムリソースに含めることができます。
+
+例えば、次のClusterRoleでは、「admin」と「edit」のデフォルトのRoleでCronTabという名前のカスタムリソースを管理できますが、「view」のRoleではCronTabリソースに対して読み取りアクションのみを実行できます。CronTabオブジェクトは、APIサーバーから見たURLで`"crontabs"`と名前が付けられていると想定できます。
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: aggregate-cron-tabs-edit
+  labels:
+    # デフォルトRoleの「admin」と「edit」にこれらの権限を追加する。
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+rules:
+- apiGroups: ["stable.example.com"]
+  resources: ["crontabs"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aggregate-cron-tabs-view
+  labels:
+    # デフォルトRoleの「view」にこれらの権限を追加します。
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+- apiGroups: ["stable.example.com"]
+  resources: ["crontabs"]
+  verbs: ["get", "list", "watch"]
+```
+
+#### Roleの例
+
+次の例は、RoleオブジェクトまたはClusterRoleオブジェクトからの抜粋であり、`rules`セクションのみを示しています。
+
+`"pods"`の読み取りを許可する
+{{< glossary_tooltip text="API Group" term_id="api-group" >}}。
+
+```yaml
+rules:
+- apiGroups: [""]
+  #
+  # HTTPレベルでの、Podにアクセスするリソースの名前
+  # オブジェクトは"pods"
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+```
+
+APIグループ`" extensions "`と `" apps "` の両方で、Deploymentsへの読み取り/書き込みを許可します。
+(HTTPレベルでURLのリソース部分に`"deployments"`を持つオブジェクトで)
+
+```yaml
+rules:
+- apiGroups: ["extensions", "apps"]
+  #
+  # HTTPレベルでの、Deploymentにアクセスするリソースの名前
+  # オブジェクトは"deployments"
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+```
+
+コアAPIグループのPodの読み取り、および`"batch"`または`"extensions"`APIグループのJobリソースの読み取りまたは書き込みを許可します。
+
+Allow reading Pods in the core API group, as well as reading or writing Job
+resources in the `"batch"` or `"extensions"` API groups:
```

Comment:
:information_source: This text should also be translated, I think you might have omitted it by accident.

Reply:
Right, I missed to remove the original text and I actually translated above the phrase.